elyra/pipeline/processor_kfp.py

```diff
@@ -169,6 +79,126 @@ def cc_pipeline():
 
         return None
 
+    def export(self, pipeline):
+        if pipeline.file_type not in ["tgz", "tar.gz", "zip", "yaml", "yml", "py"]:
+            self.log.warn("Pipeline file type not recognized...defaulting to tar.gz")
+            pipeline_file_ext = "tar.gz"
+        else:
+            pipeline_file_ext = pipeline.file_type
+
+        timestamp = datetime.now().strftime("%m%d%H%M%S")
+        pipeline_name = (pipeline.title if pipeline.title else 'pipeline') + '-' + timestamp
+
+        full_path_to_pipeline = os.getcwd() + '/' + pipeline_name + '.' + pipeline_file_ext
+
+        if pipeline_file_ext != "py":
+            try:
+                pipeline_function = lambda: self._cc_pipeline(pipeline, pipeline_name)  # nopep8
+                kfp.compiler.Compiler().compile(pipeline_function, full_path_to_pipeline)
+            except Exception as ex:
+                raise RuntimeError('Error compiling pipeline {} at {}'.
```

Comment:
Although the traceback may be part of this, I'd prefer we differentiate error messages.  This is currently a duplicate of the process() method.  How about: 'Error compiling pipeline {} for export at {}' or 'Error exporting pipeline {} at {}' if the result of the compilation is the exported file anyway.